### PR TITLE
adding jamulus-server.desktop and replacing old icon with 2020 svg

### PR DESCRIPTION
**Pull request by jujudusud**
Discussing here #1316 about adding a Jamulus server desktop file.
Done with merging this PR.

Adding (and replacing the older) icon by the two new light svg icons of 2020.

### CODE REVIEW

File: distributions/jamulus-server.desktop.in [hunk 1]
```diff
@@ -0,0 +1,18 @@
+[Desktop Entry]
+Name=Jamulus (server)
```
Comment (trebmuh) [inline distributions/jamulus-server.desktop.in:2]: We should add `Name[fr]=Jamulus (serveur)` here.

Comment (jujudusud) [reply distributions/jamulus-server.desktop.in:2]: done 69cf722

**Comment by ann0see** [comment]
Not sure what's wrong, but the CI failed. Could you please look at the checks page?

**Comment by jujudusud** [comment]
@trebmuh , do you want to update debian build part now ? 
The problem is that I removed 'jamulus.png' file from the repository. 
I will juste change something in the file ...

**Comment by jujudusud** [comment]
Please merge this one, It is necessary for packagers.
when it is merged, there will still be some work to do to ensure the future version 3.8.0.

**Comment by hoffie** [comment]
Sorry, I've somehow missed this. What is the reason for the rather long file name(s) `jamulus-icon-2020.svg` and `jamulus-server-icon-2020.svg` while the existing PNG-based icon is just `jamulus.png`?

**Comment by jujudusud** [comment]
> What is the reason for the rather long file name(s) `jamulus-icon-2020.svg` and `jamulus-server-icon-2020.svg` while the existing PNG-based icon is just `jamulus.png`?

It is just because it is a copy of the original file of the New look of Jamulus (2020). Before this New look, thé file was Jamulus.png
I can't delete jamulus.png because it is part Debian package.

**Comment by pljones** [comment]
It looks like there's something seriously wrong here, with the git history.

**Comment by jujudusud** [comment]
Unable to solve conflict. closing here.

**Pull request closed without merging**